src/test/java/uk/gov/hmcts/reform/coh/controller/OnlineHearingControllerTest.java

```diff
@@ -123,7 +125,12 @@ public void testCreateOnlineHearing() throws Exception {
                 .andReturn();
 
         CreateOnlineHearingResponse response = (CreateOnlineHearingResponse) JsonUtils.toObjectFromJson(result.getResponse().getContentAsString(), CreateOnlineHearingResponse.class);
-        assertEquals(uuid.toString(), response.getOnlineHearingId());
+        try {
+            UUID.fromString(response.getOnlineHearingId());
+            assertTrue(true);
+        } catch (Exception e) {
+            assertTrue(false);
+        }
     }
```

Comment:
Add a test to make sure the onlinehearinghistory object is added to the onlinehearing after calling the registerStateChange() method.